src/transport/FabricTable.h

```diff
@@ -41,45 +41,44 @@
 namespace chip {
 namespace Transport {
 
-typedef uint16_t AdminId;
-static constexpr AdminId kUndefinedAdminId            = UINT16_MAX;
+static constexpr FabricIndex kUndefinedFabricIndex    = UINT8_MAX;
 static constexpr uint8_t kFabricLabelMaxLengthInBytes = 32;
 
 // KVS store is sensitive to length of key strings, based on the underlying
 // platform. Keeping them short.
-constexpr char kAdminTableKeyPrefix[] = "CHIPAdmin";
-constexpr char kAdminTableCountKey[]  = "CHIPAdminNextId";
+constexpr char kFabricTableKeyPrefix[] = "CHIPAdmin";
+constexpr char kFabricTableCountKey[]  = "CHIPAdminNextId";
```

Comment:
Suggest rename `CHIPAdmin` key to `Fabrics`. Furthermore, `CHIPAdminNextId` could be renamed `NumFabrics`